src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -34,6 +34,9 @@ def load_arguments(self, _):
         c.argument('infrastructure', options_list=['--infrastructure'], help='The infrastructure on which the Kubernetes cluster represented by this connected cluster will be running on.', arg_type=get_enum_type(Infrastructure_Enum_Values))
         c.argument('disable_auto_upgrade', options_list=['--disable-auto-upgrade'], action='store_true', help='Flag to disable auto upgrade of arc agents.')
         c.argument('cl_oid', options_list=['--custom-locations-oid'], help="OID of 'custom-locations' app")
+        c.argument('container_registry_repository', options_list=['--registry-repository'], help='Custom container registry repository.')
```

Comment:
Would be kind of good to add examples in help.py which shows the customers how to onboard a cluster using pcr
https://github.com/Azure/azure-cli-extensions/blob/45873317506bc21d240de721559363b34ec7b6ee/src/connectedk8s/azext_connectedk8s/_help.py#L29

Reply:
Ack